readthedocs/templates/account/signup.html

```diff
@@ -37,7 +37,7 @@ <h1>{% trans "Sign Up" %}</h1>
     {% endblocktrans %}
 </p>
 {% else %}
-<form class="signup" id="signup_form" method="post" action="{% url 'account_signup' %}">
+<form class="signup" id="signup_form" method="post" style="width: 60%;" action="{% url 'account_signup' %}">
```

Comment:
probably we can delete this now? I think with the new templates coming, we are avoiding inline css.

Reply:
True, just checked. Will remove it.

Reply:
https://github.com/readthedocs/readthedocs.org/pull/8564